content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "End-to-End Pipeline Deployment on Azure"
```

Comment:
The title of the doc is a little too similar to the "Deployment" doc that immediately precedes it in the left-hand nav. Readers may become confused, not knowhing which guide to follow. How about "End-to-end Pipeline Example on Azure".